src/clims/models/workunit.py

```diff
@@ -0,0 +1,80 @@
+from __future__ import absolute_import
+
+from django.db import models
+from sentry.db.models import (FlexibleForeignKey, sane_repr)
+from clims.models.substance import Substance
+from clims.models.container import Container
+
+
+class WorkUnit(models.Model):
+    """
+    Represents a single item that needs to be processed manually. A WorkBatch consists of 1..n
+    WorkUnits.
```

Comment:
I'm not sure what you mean with "processed manually". I guess you refer to the real world here?

Reply:
That means that a human will have to work on it, rather than something that happens automatically within a workflow process.

Reply:
ok